### PR TITLE
refactor(design-system): replace functional components

### PR DESCRIPTION
**Pull request by mutdmour**

### CODE REVIEW

**Review by alexgrozav — COMMENTED** [review]
You're a hero, @mutdmour!
 
Only concern left is the human readable classes that are targeted from place to place in the editor-ui. I know for sure that `.button` and `.callout` are used. Did you double check them all before renaming with the `n8n-` prefix?

File: packages/design-system/src/components/N8nButton/Button.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
 	<button
-		:class="classes"
+		:class="['n8n-button', ...classes]"
```
[the remark this answers is not in the document]
Comment (mutdmour) [reply packages/design-system/src/components/N8nButton/Button.vue:3]: great points.. sorry I guess I rushed through this.. will go back and double check

**Review by alexgrozav — APPROVED** [review]
Awesome work! Tested it out, looks good to me.

**Comment by janober** [comment]
Got released with `n8n@0.190.0`

**Pull request merged**